buildkite/graphql/team_member.graphql

```diff
@@ -0,0 +1,60 @@
+fragment TeamMemberValues on TeamMember {
+    id
+    uuid
+    team {
+        id
+    }
+    user {
+        id
+    }
+    role 
+}
+
+mutation createTeamMember(
+    $teamID: ID!, 
+    $userID: ID!,
+    # @genqlient(omitempty: true)
```

Comment:
Whats the reasoning behind omitempty instead of pointer: true?

Reply:
More to get around the eventual case that we can omit it entirely (and then the API should create said TM with the default role for the team) - more below (and also an improvement case since the older resource had the role as required (not optional) 👍